.env.exmaple

```diff
@@ -0,0 +1,2 @@
+BIGCOMMERCE_STOREFRONT_API_TOKEN=
```

Comment:
@Vinsanity Mind fixing the typo in this filename? After that, we're good to merge.